packages/dd-trace/test/appsec/stack_trace.spec.js

```diff
@@ -0,0 +1,330 @@
+'use strict'
+
+const { assert } = require('chai')
+const path = require('path')
+
+const {
+  filterOutFramesFromLibrary,
+  reportStackTrace
+} = require('../../src/appsec/stack_trace')
+
+describe('Stack trace reporter', () => {
+  describe('filterOutFramesFromLibrary', () => {
+    it('should filer out frames from library', () => {
+      const callSiteList =
+        [...Array(10).keys()].map(i => (
```

Comment:
why do you need th .keys() ?
```suggestion
        Array(10).map((,i) => (
```

Reply:
`map` callback is invoked only for array indexes which have assigned values. [Doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#description)

The way you propose creates a 10-lengthed empty array, so map callback won't be called.

Reply:
ah yes i forgot you need .fill() before